cassdegrees/api/models.py

```diff
@@ -69,3 +69,8 @@ class ProgramModel(models.Model):
 
     class Meta:
         unique_together = (("code", "year"), ("name", "year"))
+
+
+class StudentModel(models.Model):
```

Comment:
Are we storing this in the main database? From what I understood, this wasn't going to be the case?

Reply:
removed